src/script/components/Modals/GroupCreation/GroupCreationModal.tsx

```diff
@@ -216,7 +220,12 @@ const GroupCreationModal: React.FC<GroupCreationModalProps> = ({
           },
         );
         setIsShown(false);
-        amplify.publish(WebAppEvents.CONVERSATION.SHOW, conversationEntity, {});
+
+        if ('key' in event) {
```

Comment:
I think you could use 
isKey from Util/KeyboardUtil 
🤔

Reply:
or maybe let's add a `isKeyboardEvent` to `KeyboardUtil`

Reply:
`isKeyboardEvent` sounds good to me, will create it and use it in all the places we do `('key' in event)` 🫡